docs/guidance/aws-eks-gpu-cluster.md

```diff
@@ -0,0 +1,74 @@
+# Start Amazon EKS Cluster with GPUs for KubeRay
+
+## Step 1: Create a Kubernetes cluster on Amazon EKS
+
+Follow the first two steps in [this AWS documentation](https://docs.aws.amazon.com/eks/latest/userguide/getting-started-console.html#)
+to: (1) Create your Amazon EKS cluster (2) Configure your computer to communicate with your cluster.
+
+## Step 2: Create node groups for the Amazon EKS cluster
+
+You can follow "Step 3: Create nodes" in [this AWS documentation](https://docs.aws.amazon.com/eks/latest/userguide/getting-started-console.html#) to create node groups. The following section provides more detailed information.
```

Comment:
Generally, should Pods be capitalized or lowercase?

Reply:
Kubernetes's website uses "Pod".